### PR TITLE
Improvements to elixirc command

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: bin/elixirc.bat [hunk 1]
```diff
@@ -15,11 +15,14 @@ goto run
...
+echo   --ignore-module-conflict    Does not emit warnings if a module was previously defined
 echo   --no-debug-info             Does not attach debug info to compiled modules
+echo   --no-docs                   Does not attach documentation to compiled modules
 echo   --verbose                   Prints compilation status
+echo   --version, -v               Prints Elixir version and exits
 echo   --warnings-as-errors        Treats warnings as errors and returns non-zero exit code
-echo   --ignore-module-conflict    Does not emit warnings if a module was previously defined
 echo.
 echo ** Options given after -- are passed down to the executed code
 echo ** Options can be passed to the Erlang runtime using ELIXIR_ERL_OPTIONS
```
Comment (eksperimental) [inline bin/elixirc.bat]: Question, can we use the "--erl" option here as in the `elixir` command?

Comment (josevalim) [inline bin/elixirc.bat]: Yes, it accepts all of elixir switches and a couple more.

Comment (eksperimental) [inline bin/elixirc.bat]: so should we document them?

Comment (josevalim) [inline bin/elixirc.bat]: It is mentioned in the header under [elixir switches], we can make it clearer if necessary but I wouldn't copy them over.

Comment (eksperimental) [inline bin/elixirc.bat]: Sorry but where are these ELIXIR_ERL_OPTIONS documented?

**Comment by josevalim** [comment]
Those are options accepted by the erl command line.
-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

**Comment by eksperimental** [comment]
They are neither documented, nor linked to the erl documentation
should we add a reference link to http://erlang.org/doc/man/erl.html
?

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**